docs/auto_arpa.md

```diff
@@ -14,16 +14,28 @@ Alternatively the value can be a dictionary with configuration options for the A
 ---
 manager:
   auto_arpa:
+    # Replace duplicate records rather than throw an error, default is False
+    # which throws an error
+    replace: False
```

Comment:
It would be better to offer a choice whether the config takes precedence or auto-generated zone. Perhaps something to do with the order of sources?

Reply:
Yeah. `YamlProvider` supports the replace idea as well so that would be better managed by flipping the order rather than having to add further configuration to all the providers involved.

That did actually make me look at the naming in `YamlProvider` and realize that it was `populate_should_replace` so I've updated this PR to use the same naming here for consistency's sake.

Reply:
That doesn't really seem directly related to auto-arpa as it would apply to situations with YamlProvider as well so that assumption was that it'd be documented in another location that was more generally applicable once that's figured out. I have hopes to spend time on documentation/examples in the ~future.